app/avo/actions/pre_made_action_batch_delete.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Actions
+  class PreMadeActionBatchDelete < Avo::BaseAction
+    self.name = I18n.t('avo.delete').capitalize
+    self.message = I18n.t('avo.are_you_sure').capitalize
+    self.confirm_button_label = I18n.t('avo.delete').capitalize
+
+    def handle(**args)
+      models, = args.values_at(:models, :fields, :current_user, :resource)
+
+      models.each(&:destroy!)
```

Comment:
This should probably be wrapped into transaction

Reply:
Actually i thanked about 3 approaches and i'm not sure which one is the better one.

1 - As you suggested wrap the delete into transaction and if one fails none is deleted
2 - Output the exception to the user for each model that fails 
3 - Delete all and stop when one of them fails

I think the better one is the one you suggested, @adrianthedev what do you think?

Reply:
I'm now thinking about exposing this action somehow to the public as it is and allowing the developer to choose the strategy.

I mean, when you run `avo:install` to generate this action in the Rails app and then enable the developer to add it to their resources and modify it as they wish.

The downside is that we should add the `index_controls` option too.